components/locale.js

```diff
@@ -0,0 +1,55 @@
+import React, {Component} from 'react'
```

Comment:
Can we add some tests for this file please? CodeCov is saying it's below threshold.